product_profile_example/profile.py

```diff
@@ -37,7 +20,7 @@ def _get_types(self):
     purchase_ok = fields.Boolean(
         string='Can be Purchased')
     available_in_pos = fields.Boolean()
-    route_ids = fields.Many2many(
+    profile_default_route_ids = fields.Many2many(
```

Comment:
Here is just in module example. It's not a module that can be used in production.
No risk here.